### ISSUE TITLE
# of characters is not correct

### ISSUE THREAD

**Issue by brichis** [Giveth/giveth-dapps-v2#2191]
**Describe the bug**
The total amount of characters is not right
![photo_2023-02-15_06-22-29](https://user-images.githubusercontent.com/106570342/219111879-46b6cebb-4fea-4189-bd2a-aa2ae2178546.jpg)

Griff asked me to create this issue, add the screenshot and tag you @MoeNick. Thank you!

**Comment by MohammadPCh** [comment]
Thanks @brichis for reporting the problem, I tried to type it in our editor,
this is the content:
One week has passed since the earthquake hit Turkey on February 6. People have been sending us donations wholeheartedly, and we are more than thankful for every penny sent to support the people and animals in need across the affected region. 



We have updated our roadmap as follows: 

Stream payments to the NGOs every 2 weeks.
Join local teams
Continue fundraising


As part of the updated roadmap, we are planning to allocate our funds proportionally based on the developments and needs on the ground 

Hold to be used in emergency situations
Transfer to Ahbap, HAYTAP and Irlaptiiiso NGOs
Support survivors migrating to other regions of Turkey.
Build an apprentice program in the DAO ecosystem, incentivise students taking on beginner jobs or small tasks in DAOs 


Read the full update [here](https://www.figma.com/file/nVoinu0tgJ565enN5R4WDE/GIVeconomy?node-id=13925-253772&t=tRdOyMET0RYkywqI-0)

and this is the result:

![image](https://user-images.githubusercontent.com/17097576/226392307-29955606-d3b4-442f-8349-9d80bb8a67b0.png)
maybe it happened because of pasting it from somewhere that has hidden characters. if you have the content I can investigate more.
